Description: Make plugins not depend on libgcc_s_sjlj-1.dll nor
 libdtdc++-6.dll.
 .
 Plugins must not depend on a shared libgcc. Newer MinGW GCC builds now also
 default to a shared libstdc++, so plugins now also require -static-libstdc++.
 .
  Without this patch, C++ plugins such as NSISdl fail to load-
Bug-RedHat: https://bugzilla.redhat.com/show_bug.cgi?id=734905
Bug-RedHat: https://bugzilla.redhat.com/show_bug.cgi?id=553971
Bug-Debian: http://bugs.debian.org/654380
Author: Kevin Kofler <Kevin@tigcc.ticalc.org>
Last-Update: 2012-01-03
--- a/SCons/Config/gnu
+++ b/SCons/Config/gnu
@@ -115,6 +115,8 @@
 plugin_env.Append(LINKFLAGS = ['-mwindows'])        # build windows executables
 plugin_env.Append(LINKFLAGS = ['$ALIGN_FLAG'])      # 512 bytes align
 plugin_env.Append(LINKFLAGS = ['$MAP_FLAG'])        # generate map file
+plugin_env.Append(LINKFLAGS = ['-static-libgcc'])   # Make sure libgcc is statically linked in, for the plugins to work.
+plugin_env.Append(LINKFLAGS = ['-static-libstdc++']) # Make sure libstdc++ is statically linked in, for the plugins to work.
 
 ### cross-platform util environment
 
